CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Change Log
+## [1.0.2]
+- Rename src and dst to rx_id and tx_id to avoid confusion
+- Bump nix dependency to `0.26`
+- Bump bitflags dependency to `2.3`
+- Bump embedded-can dependency to `0.4`
```

Comment:
nit: empty line here please.

Reply:
I added an empty line after the heading.